compiler-plugin/src/main/kotlin/com/google/devtools/ksp/symbol/impl/kotlin/KSValueArgumentImpl.kt

```diff
@@ -26,15 +26,22 @@ import com.google.devtools.ksp.symbol.KSVisitor
 import com.google.devtools.ksp.symbol.Location
 import com.google.devtools.ksp.symbol.NonExistLocation
 import com.google.devtools.ksp.symbol.Origin
+import java.util.Objects
 
 class KSValueArgumentLiteImpl private constructor(
     override val name: KSName,
     override val value: Any?,
-    override val parent: KSNode?
+    override val parent: KSNode?,
+    override val isDefault: Boolean
 ) : KSValueArgumentImpl() {
-    companion object : KSObjectCache<Triple<KSName, Any?, KSNode>, KSValueArgumentLiteImpl>() {
-        fun getCached(name: KSName, value: Any?, parent: KSNode) = cache
-            .getOrPut(Triple(name, value, parent)) { KSValueArgumentLiteImpl(name, value, parent) }
+    companion object : KSObjectCache<Int, KSValueArgumentLiteImpl>() {
+        private fun getKey(name: KSName, value: Any?, parent: KSNode, isDefault: Boolean): Int {
```

Comment:
I think it is ok to keep the old hashing input, they are enough to distinguish each other.

Reply:
OK